docs/fsharp/tour.md

```diff
@@ -67,7 +67,7 @@ It's important to note that because `struct` tuples are value types, they are ca
 
 Pipe operators (`|>`, `<|`, `||>`, `<||`, `|||>`, `<|||`) and composition operators (`>>` and `<<`) are used extensively when processing data in F#.  These operators are functions which allow you to establish "pipelines" of functions in a flexible manner.  The following walks though how you could take advantage of these operators to build a simple functional pipeline.
 
```

Comment:
previous line has a typo: 
The following walks though -> The following example walks through